lib/iris/tests/results/file_load/known_loaders.txt

```diff
@@ -1,16 +1,16 @@
  * NetCDF OPeNDAP (priority 6)
  * NAME III (priority 5)
- * NetCDF_v4 (priority 5)
- * UM Post Processing file (PP) (priority 5)
  * NetCDF (priority 5)
  * NetCDF 64 bit offset format (priority 5)
+ * NetCDF_v4 (priority 5)
+ * UM Post Processing file (PP) (priority 5)
```

Comment:
Perhaps the simplest solution is to tweak the definitions in `lib/iris/fileformats/__init__.py` to use unique numbers that preserve the current order?